submissions/loki13btz/html-css-popup/index.html

```diff
@@ -0,0 +1,122 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./style.css">
+    <title>Hooli-style Popup</title>
+</head>
+
+<body>
+    <header class="header">
+        <nav class="header-navigation">
+            <ul class="menu-list">
+                <li><a href="#" class="menu-item">Mail</a></li>
+                <li><a href="#" class="menu-item">Images</a></li>
+                <li><label for="popupCheckbox" class="menu-item popup-shower"><img class="popup-icon"
```

Comment:
Add `cursor: pointer` to all interactive elements.
Also, the user must have the ability to open a popup using keyboard.